test/kubectl/kubectl.go

```diff
@@ -97,6 +98,9 @@ func DeleteByName(opts Options, kind, name string, ignoreNotFound bool) error {
 	if ignoreNotFound {
 		cmd.Args = append(cmd.Args, "--ignore-not-found")
 	}
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
```

Comment:
Why?

Reply:
Dunno 🤷 @adutra ?

Reply:
These will break the next CombinedOutput command for no reason (it sets those parameters and returns the result). Just remove these changes.

Reply:
I don't see the purpose of this change either. I think reverting is the right thing to do?

Reply:
I'll revert the changes